applications/mne_scan/plugins/neuronalconnectivity/neuronalconnectivity.cpp

```diff
@@ -103,6 +103,7 @@ NeuronalConnectivity::NeuronalConnectivity()
     AbstractMetric::m_bStorageModeIsActive = true;
     AbstractMetric::m_iNumberBinStart = 0;
     AbstractMetric::m_iNumberBinAmount = 100;
+    m_pActionShowYourWidget = nullptr;
```

Comment:
Use Q_NULLPTR. See [here](https://stackoverflow.com/questions/36280910/when-to-use-q-nullptr).